pkgs/sagemath-doc-html/meson.build

```diff
@@ -0,0 +1,21 @@
+project('sagemath-doc-html',
+        version: run_command('cat',
+                             join_paths(meson.current_source_dir(), 'VERSION.txt'),
+                             check: true).stdout().strip()
+)
+
+docdir = get_option('datadir') / 'share' / 'doc' / 'sage'
+
+html = custom_target('HTML documentation',
+  output: 'index.html',
+  command: [join_paths(meson.current_source_dir(), 'doc', 'make.sh'),
+            '-C', join_paths(meson.current_source_dir(), 'doc'), 'doc-html'],
+  env: {'SAGE_DOC': meson.current_build_dir()},
+  install: true,
+  install_dir: docdir
+)
+
+install_subdir(meson.current_build_dir() / 'html',
+               install_dir: docdir)
+
+meson.add_dist_script('meson_dist_script.sh')
```

Comment:
One of the nice things about meson is that it also works on windows. So, to not unnecessarily break the cross-platform compatibility, please convert the script to python.

Reply:
Sure, can do

Reply:
... in a follow-up PR when any of this can actually be tested on Windows.

Reply:
Can you please do it as part of this PR, thanks!

Reply:
Perhaps explain where any urgency of this would come from

Reply:
No urgency, just doing it right from the start.

Reply:
I already told you at what point I'll take care of it. I've opened #36800 for it so it won't be forgotten.

Reply:
Why not do it now? In fact, I think it would be better to not create the symlinks in the first place (ie introduce an option in the docbuild to just copy the files instead). Would also help later with the ci.

Reply:
Sounds like you need to look up what `add_dist_script` does.

Reply:
?